### DISCUSSION TITLE
Create user and login error...

### DISCUSSION THREAD

**Discussion by gennai3** [SonicJs-Org/sonicjs#124] [Q&A] [answered]
Hi,
I use SonicJS Beta 0.5.4  today, I made user account at Admin enviroment /admin/users, however , I can not view new account name at /admin/users.

Also I create a user by same name and same password at /register page. I can careat this my account now. 

And I checked SonicJS admin enviroment by another login user account, I can view new account name at /admin/users page now.
However, eidt this new account, next page is /admin/user/edit/9c9f046c-5978-4f76-9019-1468f67449b8    "Nginx 502 Bad Gateway error "

And login by new account name and password, I had error message now at /user  page, "{"user":{"id":"9c9f046c-5978-4f76-9019-1468f67449b8","username":"hodota@genxxxxx","profile":{},"createdOn":"2022-04-12T08:22:41.985Z","updatedOn":"2022-04-12T08:22:41.985Z"}}"
However, I can login to site and I use admin enviromrnt now.  I think that this enviroment is not create role functions now, it is problems or something that. 

Please let me, thanks.

Cheers,
Kazu Hodota

**Comment by lane711** [comment, marked as answer]
Hi Kazu, sorry for the delay. This issue should be resolved if you get the latest on the main branch.

**Reply by gennai3** [reply]
Thank you, I am going to learn SonicJS at own site, https://sonicjs.cmslabo.co.jp  now.
